src/database/index.tsx

```diff
@@ -182,17 +182,34 @@ export const getBaselineScenarioRef = async (): Promise<firebase.firestore.Docum
   return results.docs[0].ref;
 };
 
-export const getScenario = async (
+const getScenarioRef = async (
```

Comment:
Anything mentioning `Ref` (i.e. a Firestore Document Reference) should now be "private" and no leak outside of this DB file.